Project.toml

```diff
@@ -1,11 +1,10 @@
 name = "MatrixDepot"
 uuid = "b51810bb-c9f3-55da-ae3c-350fc1fbce05"
-version = "1.0.1"
+version = "1.0.2"
```

Comment:
It's generally easier to manage if versions are bumped separately from the PR.

Reply:
understood